tutorials/vector_point/index.pug

```diff
@@ -0,0 +1,56 @@
+extends ../common/index.pug
+
+block mainTutorial
+  :markdown-it
+    # Tutorial - Vector point
+    First, let's create our map and add a base map. Also, to make simple, we import our data inline.
+  
+  +codeblock('javascript', 1, undefined, true).
+    var map = geo.map({
+      node: "#map",
+      center: {x: -97.67, y: 31.80},
+      zoom: 4
+    });
+    map.createLayer('osm');
+
+    var cities = {"type":"FeatureCollection","features":[{"type":"Feature","geometry":{"type":"Point","coordinates":[-74.0059413,40.7127837]},"properties":{"name":"New York","population":"8405837"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-118.2436849,34.0522342]},"properties":{"name":"Los Angeles","population":"3884307"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-87.6297982,41.8781136]},"properties":{"name":"Chicago","population":"2718782"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-95.3698028,29.7604267]},"properties":{"name":"Houston","population":"2195914"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-75.1652215,39.9525839]},"properties":{"name":"Philadelphia","population":"1553165"}},{"type":"Feature","geometry":{"type":"Point","coordinates":[-112.0740373,33.4483771]},"properties":{"name":"Phoenix","population":"1513367"}}]};
+
+  :markdown-it
+    We create a layer first.
+    
+  +codeblock('javascript', 2).
+    var layer = map.createLayer('feature'); 
+
+
+  :markdown-it
+    Then create point feature and set the data.
+
+  +codeblock('javascript', 3).
+    var feature = layer.createFeature('point')
+      .data(cities.features)
+      .position(function (city) {
+          return { x: city['geometry']['coordinates'][0], y: city['geometry']['coordinates'][1] };
```

Comment:
I'd recommend, in general, using shorter lines in the tutorial so that there is less chance the user has to scroll to see the whole line.  So this return could be rewritten as
```
        return {
          x: city['geometry']['coordinates'][0], 
          y: city['geometry']['coordinates'][1] 
        };
```